frog-utils/src/generateReactiveMem.js

```diff
@@ -81,26 +82,9 @@ export class MemDoc {
     return path.reduce((acc, x) => acc[[x]], data);
   }
 
-  setMaxListeners = () => {};
-
-  removeListener = () => {
-    this.callback = () => {};
-  };
-
-  subscribe = () => {};
-
-  destroy = () => {
-    this.callback = () => {};
-  };
-
-  on = (type: string, callback: Function) => {
-    if (type === 'op') {
-      this.callback = callback;
-    }
-    if (type === 'load') {
-      callback();
-    }
-  };
+  destroy() {}
+  setMaxListeners() {}
+  removeListener() {}
```

Comment:
As you use a specific EventEmitter type. Does it still make sense to override the `removeListener` method?